packages/cloud-cognitive/src/components/NonLinearReading/_non-linear-reading.scss

```diff
@@ -0,0 +1,156 @@
+//
+// Copyright IBM Corp. 2023, 2023
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// Standard imports.
+@import '../../global/styles/project-settings';
+
+@mixin term-closed-defaults {
+  // stylelint-disable-next-line carbon/layout-token-use
+  padding: 1px $spacing-03 1px $spacing-03;
+  border: none;
+  border-radius: 100px;
+  background-color: $interactive-02;
+  white-space: nowrap;
+
+  &:hover {
+    background-color: $hover-secondary;
+    cursor: pointer;
+  }
+}
+@mixin term-open-defaults {
+  @include term-closed-defaults();
+
+  padding-right: 0;
+
+  svg {
+    height: 18px;
```

Comment:
Was going to recommend

```suggestion
    height: rem(18px);
```
But upon closer inspection, doesn’t seem like this line takes effect right now anyway?

![image](https://user-images.githubusercontent.com/1717398/228102625-c3899272-ab59-492f-94f3-2f05b294e83b.png)

Reply:
That's weird. Technically that tooltip information is correct. And yet turning that prop on and off, or increasing the height, actually effects a change.

I did some testing, and it turned to be even weirder:
![stylelint-errors](https://user-images.githubusercontent.com/52505287/228643843-a9e95702-d643-4864-a5d2-4ef10d42c88c.png)

Reply:
I would guess that border is excluded from the the lint rules since a `1px` border is pretty common to add. I think our linting flags these because we prefer to follow the spacing patterns. A lot of our components are based on composing with others. NLR is more of a newly constructed one so I’m not too surprised by this? There may be less need to flag/disable in the other n2p components?